src/renderers/shaders/ShaderLib/meshphysical_frag.glsl.js

```diff
@@ -100,6 +100,9 @@ void main() {
 	#include <clearcoat_normal_fragment_begin>
 	#include <clearcoat_normal_fragment_maps>
 	#include <emissivemap_fragment>
+
+	vec3 rawDiffuseColor = diffuseColor.rgb;
```

Comment:
Let me know if you have any better and non-confising name idea than `rawDiffuseColor`.

Reply:
`baseDiffuseColor` maybe?

We can change that later, I'll merge this for now.